Content.Server/GameObjects/Components/BaseComputerUserInterfaceComponent.cs

```diff
@@ -0,0 +1,126 @@
+using Content.Server.GameObjects.Components.Construction;
+using Content.Server.GameObjects.Components.Power.ApcNetComponents;
+using Content.Server.Utility;
+using Content.Shared.GameObjects.Components;
+using Content.Shared.GameObjects.EntitySystems;
+using Content.Shared.GameObjects.EntitySystems.ActionBlocker;
+using Content.Shared.Interfaces;
+using Content.Shared.Interfaces.GameObjects.Components;
+using Robust.Server.GameObjects;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.ViewVariables;
+using Robust.Shared.Localization;
+
+namespace Content.Server.GameObjects.Components
+{
+    /// <summary>
+    /// This component is used as a base class for classes like SolarControlConsoleComponent.
+    /// These components operate the server-side logic for the "primary UI" of a computer.
+    /// That means showing the UI when a user activates it, for example.
+    /// </summary>
+    [ComponentReference(typeof(IActivate))]
+    public abstract class BaseComputerUserInterfaceComponent : Component, IActivate
+    {
+        // { get; private set; } doesn't really express this properly.
```

Comment:
Remove?

Reply:
Remove the comment, or?

Reply:
Yeah, if the comment isn't needed then remove it. It doesn't look like it's needed.

Reply:
Done